sirepo/package_data/static/js/sirepo-command.js

```diff
@@ -82,6 +82,15 @@ SIREPO.app.factory('commandService', function(appState, latticeService, panelSta
         return value;
     };
 
+    self.getNextCommand = (type) => {
```

Comment:
I think we are omitting parens for single parameters:
```javascript
self.getNextCommand = type => {...};
```

Reply:
77045d74288ceb536b10500920e635863ad3d399